website/docs/cloud-docs/api-docs/audit-trails.mdx

```diff
@@ -108,23 +108,327 @@ $ curl \
+
+| Key                        | Data Type | Description                                             |
+| ---                        | ---       | ---                                                     |
+| `delete_older_than_n_days` | integer   | The number of days to retain data before deletion. Must be greater than 0. |
+| `organization`             | string    | The organization associated with the data retention policy. |
+| `organization_id`          | string    | The ID of the organization associated with the data retention policy. |
+
+### Project events
+
+[Projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) let you organize your workspaces and scope access to workspace resources.
+
+#### Add Team
+
+A Terraform Cloud organization emits this event when someone in your organization adds a team(`add_team`) to a project. Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
+
+| Key          | Data Type | Description                           |
+| ---          | ---       | ---                                   |
+| `team`       | string    | The name or identifier of the team.    |
+| `permissions` | string   | The permissions granted to the team.  |
+
+#### Update Team Permissions
+
+A Terraform Cloud organization emits this event when someone in your organization updates the permissions(`update_team_permissions`) of a team on a project. Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
+
+| Key          | Data Type | Description                           |
+| ---          | ---       | ---                                   |
+| `team`       | string    | The name or identifier of the team.    |
+| `permissions` | string   | The updated permissions for the team. |
+
+#### Remove Team
+
+A Terraform Cloud organization emits this event when someone in your organization removes a team(`remove_team`) from a project. Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
+
+| Key          | Data Type | Description                           |
+| ---          | ---       | ---                                   |
+| `team`       | string    | The name or identifier of the team.    |
+
+### Policy checks
+
+[Policy checks run Sentinel policies](/terraform/enterprise/policy-enforcement/manage-policy-sets). 
+
+#### Process audit logs
+
+A Terraform Cloud organization emits this event when processing an audit log event related to policy checks(`process_audit_log_event`). Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
```

Comment:
```suggestion
A Terraform Cloud organization emits this event when processing an audit log event related to policy checks. Alongside the [standard audit trail fields](#standard-response-fields), this event response includes the following fields:
```

I believe we can remove this piece of information which is internal to the code. cc @carolinaborim

Reply:
agreed!